internal/xdscache/v3/listener.go

```diff
@@ -288,14 +288,24 @@ type ListenerCache struct {
 }
 
 // NewListenerCache returns an instance of a ListenerCache
-func NewListenerCache(config ListenerConfig, address string, port int) *ListenerCache {
-	stats := envoy_v3.StatsListener(address, port)
-	return &ListenerCache{
+func NewListenerCache(config ListenerConfig, statsAddress string, statsPort, adminPort int) *ListenerCache {
+	stats := envoy_v3.StatsListener(statsAddress, statsPort)
+	admin := envoy_v3.AdminListener("127.0.0.1", adminPort)
```

Comment:
Should we set this to the socket name to make sure there's no confusion, or is this only used if we're exposing the admin back on a network interface?

Reply:
This is used to create a listener which exposes the read-only bits from the admin page.